assets/csidriveroperators/openstack-cinder/hypershift/guest/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+resources:
+  - ../../base
+namespace: openshift-cluster-csi-drivers
+patches:
+  # TODO(stephenfin): Other operators patch in access to persistentclaimvolumes here,
+  # but we allow that in our base asset (meaning, in both a standalone and a HCP
+  # guest cluster). Why the distinction? Should we be stricter?
+  - path: monitoring_role.patch.yaml
```

Comment:
Yeah, I don't think you need permissions to `get`, `list` PVCs in hypershift's mgmt namespace. That will be strange, so we do need to just patch it.

Reply:
Sweet, done.